### PR TITLE
Add Optimism bridge box blog post

### PR DESCRIPTION
**Pull request by heydamali**

### CODE REVIEW

File: src/blog/introducing-the-optimism-bridge-truffle-box/index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+3. An [L2 contract](https://github.com/truffle-box/optimism-bridge-box/blob/232a6db7784ed9131d5584f0fc16abec92b21a51/contracts/optimism/GreeterL2.sol) that sends a message over the Optism bridge.
+4. A [Migration](https://github.com/truffle-box/optimism-bridge-box/blob/232a6db7784ed9131d5584f0fc16abec92b21a51/migrations/4_set_L1_greeting.js) that sends a message from Optimism to Ethereum.
+5. A [script](https://github.com/truffle-box/optimism-bridge-box/blob/232a6db7784ed9131d5584f0fc16abec92b21a51/scripts/deploy.mjs) to automate the process of compiling contracts and running migrations across each network.
+
+## Conclusion
+That’s all you need to get started building dapps that bridge between Optimism and Ethereum. We’ve put together a short [demo](https://github.com/truffle-box/optimism-bridge-box/blob/232a6db7784ed9131d5584f0fc16abec92b21a51/README.md#demo) that further expands on how the Optimism Bridge Box can be used to bridge networks. We’re excited and can not wait to see what our developer community builds with this these tools.
```
Comment (emilyJLin95) [inline src/blog/introducing-the-optimism-bridge-truffle-box/index.md:76]:
```suggestion
That’s all you need to get started building dapps that bridge between Optimism and Ethereum. We’ve put together a short [demo](https://github.com/truffle-box/optimism-bridge-box/blob/232a6db7784ed9131d5584f0fc16abec92b21a51/README.md#demo) that further expands on how the Optimism Bridge Box can be used to bridge networks. We’re so excited to see what our developer community builds with this these tools!
```

File: src/blog/introducing-the-optimism-bridge-truffle-box/index.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+5. A [script](https://github.com/truffle-box/optimism-bridge-box/blob/232a6db7784ed9131d5584f0fc16abec92b21a51/scripts/deploy.mjs) to automate the process of compiling contracts and running migrations across each network.
+
+## Conclusion
+That’s all you need to get started building dapps that bridge between Optimism and Ethereum. We’ve put together a short [demo](https://github.com/truffle-box/optimism-bridge-box/blob/232a6db7784ed9131d5584f0fc16abec92b21a51/README.md#demo) that further expands on how the Optimism Bridge Box can be used to bridge networks. We’re excited and can not wait to see what our developer community builds with this these tools.  
+
+To find out more about our suite of developer tools, visit the official [Truffle website](https://trufflesuite.com). If you have questions that you'd like for our team members to answer, concerning the merge and how it impacts our products, feel free to start a discussion on our [Github Discussions](https://github.com/orgs/trufflesuite/discussions) channel. Finally, don't forget to follow us on [Twitter](https://twitter.com/trufflesuite) for live announcements and updates.
```
Comment (emilyJLin95) [inline src/blog/introducing-the-optimism-bridge-truffle-box/index.md:78]:
```suggestion
To find out more about our suite of developer tools, visit the official [Truffle website](https://trufflesuite.com). If you have questions that you'd like for our team members to answer, feel free to start a discussion on [Github Discussions](https://github.com/orgs/trufflesuite/discussions). Finally, don't forget to follow us on [Twitter](https://twitter.com/trufflesuite) for live announcements and updates.
```

File: src/blog/introducing-the-optimism-bridge-truffle-box/index.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+- [Node.js](https://nodejs.org/) 10.x or later
+- [NPM](https://docs.npmjs.com/cli/) 5.2 or later
+- [Docker](https://docs.docker.com/get-docker/) 19.03.12 or later
+- [Docker-compose](https://docs.docker.com/compose/install) version 1.27.3 or later
+- Recommended Docker memory allocation of >= 8 GB
+- Windows, Linux, or macOS
+
+Other important, but optional requirements are:
+- An [Infura](https://blog.infura.io/?utm_source=blog&utm_medium=pressrelease&utm_campaign=2022_Jun_introducing-the-optimism-bridge-truffle-box_acquisition_content) account and a Project ID
+- A [MetaMask](https://metamask.io/) account
```
Comment (emilyJLin95) [inline src/blog/introducing-the-optimism-bridge-truffle-box/index.md:46]: can you do the same for metamask?

**Review by leeftk — APPROVED** [review]
LGTM!!!

**Comment by kevinbluer** [comment]
LGTM too. I'll go ahead and merge now (as @heydamali wanted it live today). @emilyJLin95 we can get [this](https://github.com/trufflesuite/trufflesuite.com/pull/1260#pullrequestreview-1023991527) pushed as a separate update.

**Pull request merged**